### PR TITLE
generate booleans

### PR DESCRIPTION
**Pull request by sakari**
requires https://github.com/smartlyio/oats/pull/56 and https://github.com/smartlyio/oats-runtime/pull/5

### CODE REVIEW

File: src/generator.ts [hunk 1]
```diff
@@ -65,6 +65,12 @@ export class GenType extends fc.Arbitrary<any> {
     if (type.type === 'object') {
       return new GenObject(type.properties, type.additionalProperties);
     }
+    if (type.type === 'boolean') {
+      if (type.enum) {
+        return fc.oneof(...type.enum.map(fc.constant));
```
Comment (buttilda) [inline src/generator.ts:70]: can there be enum booleans? 🤔

File: examples/openapi.yaml [hunk 2]
```diff
@@ -20,6 +20,8 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/TestTarget'
+        booleanFlagg:
```
Comment (buttilda) [inline examples/openapi.yaml]:
```suggestion
        booleanFlag:
```

**Review by buttilda — APPROVED** [review]

File: src/generator.ts (hunk 1 above)
Comment (sakari) [reply src/generator.ts:70]: I guess. actually I think that we can have `enum: true, 2, 'xx' ` even but that is so silly that I did not want to deal with it.

Comment (buttilda) [reply src/generator.ts:70]: please lets not 😅

**Pull request merged**